trussworks/react-uswds
# feat: New Component Icon

# Summary

This PR adds the [Icon component](https://designsystem.digital.gov/components/icon/).

## Related Issues or PRs

closes #992 

## How To Test

Check out this branch and run `yarn test` to execute tests and `yarn storybook` to see the new component in action. If you cannot see the Storybook controls dashboard, press the `D` key and it should appear on the right-hand side.

Alternatively, you can scroll to the bottom of this page and click "Show environments" on the right above the comment input box.


### Screenshots


![11May-17-2021 12-01-57](https://user-images.githubusercontent.com/16230705/118542376-c902f380-b707-11eb-8108-5e7bd7858314.gif)
